src/panoramic/cli/context.py

```diff
@@ -8,3 +9,15 @@
 def get_company_slug() -> str:
     """Return company slug from context."""
     return read_yaml(Paths.context_file())['company_slug']
+
+
+@functools.lru_cache()
+def get_dbt_packages() -> Optional[List[Dict[str, Any]]]:
+    """Return dbt packages from context."""
+    return read_yaml(Paths.context_file()).get('dbt', {}).get('packages')
```

Comment:
[nitpick] `Paths.context_file()).get('dbt', {})` is repeated 3 times - it might be prettier to extract some part of it - be it `'dbt'` to a constant or the whole expression to some `Paths` scoped method.

Reply:
Can do the constant although it seems a bit unnecessary. `Paths` method wouldn't work since this is parsing of the dictionary that's in the pano.yaml file, not a path operation, right?

Reply:
Sure, I'll leave that to you then, we can always refactor if we add more usages later.